slides/src/content/bios/sonu-kapoor.md

```diff
@@ -0,0 +1,7 @@
+## Sonu Kapoor
+
+<img src="./sonu-kapoor.jpeg" alt="Sonu Kapoor" style="max-width: 200px;" />
+
+Sonu Kapoor is a JavaScript Developer at Rangle.io. 
```

Comment:
Oh c'mon - you're more interesting than that.  Give us a sentence or two about your background :-)